specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -8937,6 +8937,18 @@
         "modelAsString": true
       }
     },
+    "DeleteOption": {
```

Comment:
Please create a new [Example ](https://github.com/Azure/azure-rest-api-specs/tree/compute-2021-03/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/examples)file for this feature.